proj/hps_accel/src/blocks.cc

```diff
@@ -122,10 +125,10 @@ void LoadInput(size_t width, size_t in_channels, const int8_t* values) {
 }
 
 Vector16 GetInput() {
-  uint32_t word0 = cfu_get(REG_INPUT_0);
-  uint32_t word1 = cfu_get(REG_INPUT_1);
-  uint32_t word2 = cfu_get(REG_INPUT_2);
-  uint32_t word3 = cfu_get(REG_INPUT_3);
+  uint32_t word0 = cfu_get_sw(REG_INPUT_0);
+  uint32_t word1 = cfu_get_sw(REG_INPUT_1);
+  uint32_t word2 = cfu_get_sw(REG_INPUT_2);
+  uint32_t word3 = cfu_get_sw(REG_INPUT_3);
```

Comment:
Is it done this way because we're mixing some HW CFU and some SW CFU emulation, so we can't just use the global switch to pick one or the other?

Reply:
Yes, that's right. There is no hw version for these yet, so we pin them to the software_cfu implementation.